### PR TITLE
Keep Mouse Active

### PR DESCRIPTION
**Pull request by akshaykhoje**
# Description

This directory contains a python script to keep the mouse moving to avoid the screen going black in case you want to keep it on all the time.

## Type of change

- [ ] New feature (non-breaking change which adds functionality)

## Checklist:

- [ ] My code follows the style guidelines(Clean Code) of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have created a helpful and easy to understand `README.md`
- [ ] I have added the project meta data in the PR template.
- [ ] I have created the ``requirements.txt`` file if needed.

Category:
- [ ] Automation

Title: Keep Mouse Active

Folder: Move-Mouse

Requirements: requirements.txt

Script: move.py

Contributor: akshaykhoje

Description: Keep the screen going on to avoid it turning off.

### CODE REVIEW

**Review by avinashkranjan — CHANGES_REQUESTED** [review]
Mention the Issue number

File: Move-Mouse/move.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import pyautogui as pgui
+import time
```
Comment (JRudransh) [inline Move-Mouse/move.py:2]: Remove unused imports

File: Move-Mouse/move.py
Comment (JRudransh) [inline Move-Mouse/move.py:1]: This program is very dangerous, If user is trying to use his mouse, you are not allowing to do it. That should be handeled correctly.

What you can do:
1. Save the initial cursor position
2. When user try to use his cursor (At the time user moves the cursor) restore the cursor initial position and halt
3. Keep track on mouse or keyboard activity
4. If you dont see any activity for `x` amount of time, resume the program

**Review by JRudransh — CHANGES_REQUESTED** [review]
This program is overriding user's cursor movement. Please act upon my comment

**Comment by akshaykhoje** [comment]
> This program is overriding user's cursor movement. Please act upon my comment

I apologise for my inactivity. I have been occupied with a couple of heavy projects lately. 

I will try doing the mentioned changes as soon as possible.

Thanks for your inputs @JRudransh.

File: Move-Mouse/move.py
Comment (akshaykhoje) [reply Move-Mouse/move.py:1]: Yup will work on it.

**Pull request closed without merging**